lisa/util/__init__.py

```diff
@@ -835,3 +855,22 @@ def get_first_combination(
         results.pop()
 
     return False
+
+
+def check_panic(content: str, stage: str, log: "Logger") -> None:
+    log.debug("checking panic in serial log...")
```

Comment:
The content may not from the serial log.

Reply:
updated